htdocs/lib/MapManager.js

```diff
@@ -72,7 +72,9 @@ MapManager.prototype.process = function(e) {
                 break;
 
             case 'receiver_details':
-                $('.webrx-top-container').header().setDetails(json.value);
+                $().ready(function () { // make sure header is loaded
```

Comment:
the header is not loaded at first, so we postpone the changes for onDomReady event

Reply:
Looks ok (but will test)

Reply:
i've tested. it works for the ready event and any later calls will still work.
but do your checks.